billing/helpers.py

```diff
@@ -1,41 +1,10 @@
-from dataclasses import asdict
-from typing import List, Optional
-
 from django.conf import settings
 
-import services.sentry as sentry
 from codecov_auth.models import Owner
-from plan.constants import (
-    ENTERPRISE_CLOUD_USER_PLAN_REPRESENTATIONS,
-    FREE_PLAN_REPRESENTATIONS,
-    PR_AUTHOR_PAID_USER_PLAN_REPRESENTATIONS,
-    SENTRY_PAID_USER_PLAN_REPRESENTATIONS,
-    TEAM_PLAN_MAX_USERS,
-    TEAM_PLAN_REPRESENTATIONS,
-    PlanData,
-    TrialStatus,
-)
-from plan.service import PlanService
+from plan.constants import ENTERPRISE_CLOUD_USER_PLAN_REPRESENTATIONS
 
 
 def on_enterprise_plan(owner: Owner) -> bool:
```

Comment:
We might be able to move this file to the plan_service class and get rid of this file altogether in the future 👌